core/src/main/java/org/testcontainers/containers/PortForwardingContainer.java

```diff
@@ -60,6 +60,13 @@ public void exposeHostPort(int port) {
             getSshConnection().requestRemotePortForwarding("", port, "localhost", port);
         }
     }
+    
+    @SneakyThrows
+    public void exposeHostPort(int hostPort, int containerPort) {
+        if (exposedPorts.add(hostPort)) {
```

Comment:
should store a pair for host/container port

Reply:
done